tests/helper/helper_oc.go

```diff
@@ -430,9 +430,9 @@ func (oc *OcRunner) WaitForDCRollout(dcName string, project string, timeout time
 	session.Wait(timeout)
 }
 
-// CheckForExistence checks if the given resource exists on the cluster
-func (oc *OcRunner) CheckForExistence(resourceName, namespace string) {
-	session := CmdRunner(oc.path, "get", "routes", "--namespace", namespace)
+// CheckForExistence checks if the given resource type exists on the cluster
+func (oc *OcRunner) CheckForExistence(resourceType, namespace string) {
+	session := CmdRunner(oc.path, "get", resourceType, "--namespace", namespace)
 	Eventually(session).Should(gexec.Exit(0))
 	output := string(session.Wait().Out.Contents())
 	Expect(output).To(ContainSubstring(""))
```

Comment:
This would still not work as expected, as `oc get <resource-type>`  returns `No resources found in <namespaceName> namespace`. It never returns empty string.

Reply:
I'm wondering if we can just remove that second check? If `oc get <resource-type>` returns  `No resources found in <namespaceName> namespace`, it will also return a non-zero exit code, meaning the `Eventually(session).Should(gexec.Exit(0))` would fail.